front/app/routes/buildbots.js

```diff
@@ -9,14 +9,14 @@ function fixTimestamps(data) {
 
 export default Ember.Route.extend({
   model: function() {
-    var summary_url = ENV.apiBaseURL + 'buildbots';
+    const summary_url = ENV.apiBaseURL + 'buildbots';
     return Ember.$.getJSON(summary_url)
       .then(metrics => {
 
-        var win_buildbot_times = [];
-        var mac_buildbot_times = [];
-        var linux_buildbot_times = [];
-        var misc_buildbot_times = [];
+        const win_buildbot_times = [];
```

Comment:
So...in current JS does `const` just mean it won't be reassigned, and `var` is eligible for reassignment? I've been writing too much Rust, I instinctively used `const` like `let` and `var` like `let mut`.